cfspopcon/formulas/fusion_power/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Routines to calculate the fusion power and gain."""
+
+from . import average_fuel_ion_mass, fusion_data, fusion_gain, fusion_rates
+
+__all__ = ["average_fuel_ion_mass", "fusion_rates", "fusion_gain", "fusion_data"]
```

Comment:
similar comment as above. 
Consider directly exposing e.g. `calc_average_fuel_ion_mass` and others.